src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlTypes/SqlFileStream.cs

```diff
@@ -1,1126 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Runtime.InteropServices;
-using System.Runtime.CompilerServices;
-using System.Globalization;
-using System.IO;
-using System.Security.Permissions;
-using Microsoft.Win32.SafeHandles;
-using System.Diagnostics;
-using System.Runtime.Versioning;
-using Microsoft.Data.Common;
-using Microsoft.Data.SqlClient;
-
-namespace Microsoft.Data.SqlTypes
-{
-    /// <include file='../../../../../../../doc/snippets/Microsoft.Data.SqlTypes/SqlFileStream.xml' path='docs/members[@name="SqlFileStream"]/SqlFileStream/*' />
-    sealed public class SqlFileStream : System.IO.Stream
```

Comment:
This is a public type - why is it being deleted?

Reply:
SqlFileStream was merged in #2898, this file just wasn't removed as part of the PR and I didn't notice during review. The netfx csproj [references](https://github.com/dotnet/SqlClient/blob/main/src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj#L721) the shared folder.

Reply:
Aw man, I thought I got that :(